mlir-tensorrt/executor/include/mlir-executor-c/Runtime/Runtime.h

```diff
@@ -375,15 +386,22 @@ static inline bool mtrtRuntimeSessionIsNull(MTRT_RuntimeSession session) {
 }
 
 /// Using `session`, execute the pubic function with the specified name.
-/// The `inArgs` and `outArgs` are arrays for input arguments and destination
-/// arguments, respectively. Input arguments may be MemRefs or scalars, but
-/// destination arguments must be MemRefs.
-/// A stream may optionally be specified, otherwise pass the result of
-/// `mtrtStreamGetNull()`.
+/// The `inArgs`, `outArgs`, and `results` are arrays for input arguments and
+/// optional destination arguments and results, respectively. Input arguments
+/// may be MemRefs or scalars, but destination arguments and results must be
+/// MemRefs. If `outArgs` are present, it is expected that `results` are empty
```

Comment:
Similarly, you can have both "output arguments" and "results", just like how in C++ you can have a function like 

```
int my_func(int in, int *out);
```